public/components/page-contributors/contributors/contributors.stache

```diff
@@ -10,24 +10,24 @@
     </tr>
   </thead>
   <tbody>
-  {{#each contributors as contributor}}
+  {{# each(contributors, contributor=value) }}
     <tr>
-      <td><model-edit-property model:from="contributor" property="name" /></td>
-      <td><model-edit-property model:from="contributor" property="email" /></td>
+      <td><model-edit-property model:from="contributor" property:from="'name'" /></td>
+      <td><model-edit-property model:from="contributor" property:From="'email'" /></td>
       <td>
-        <input type="checkbox" checked:from="active" on:change="setActive(contributor, %element.checked)"{{#if isSaving}} disabled{{/if}} />
+        <input type="checkbox" checked:from="active" on:change="../setActive(contributor, scope.element.checked)"{{# if(isSaving()) }} disabled{{/ if }} />
       </td>
     </tr>
-  {{/each}}
+  {{/ each }}
   </tbody>
 </table>
 
-{{#if isAddingContributor}}
+{{# if(isAddingContributor) }}
 <div class="panel panel-default">
     <div class="panel-heading">Add Contributor</div>
     <div class="panel-body">
-      <form on:submit="addContributor(%event)" id="new-contributor">
-        {{#if newContributorError}}<div class="alert alert-danger" role="alert">{{newContributorError}}</div>{{/if}}
+      <form on:submit="addContributor(scope.event)" id="new-contributor">
```

Comment:
Instead of `%event`, use `scope.event`. All "special" values are on the scope object now.